lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -34,6 +36,15 @@ def to_native(self, course_overview):
                 kwargs={'course_id': course_id},
                 request=request
             )
+            course_overview_tabs = course_overview.course_overview_tab.all()
+            for tab in course_overview_tabs:
+                if tab.tab_id == "discussion" and is_discussion_enabled(course_id):
```

Comment:
@nasthagiri how about we move it to CourseOverview instead? the loop could also be moved there to get to know if discussion is enabled for this course_overview?